network/base/views.py

```diff
@@ -348,6 +357,20 @@ def prediction_windows(request, sat_id, start_date, end_date, station_id=None):
     for station in stations:
         if not station.online:
             continue
+
+        # skip if this station is not capable of receiving the frequency
+        frequency_supported = False
+        for gs_antenna in station.antenna.all():
+            try:
+                if (int(gs_antenna.frequency) <=
+                        int(downlink) <=
+                        int(gs_antenna.frequency_max)):
+                    frequency_supported = True
+            except TypeError:
```

Comment:
they are not required though, it is possible to end up with a blank downlink_low.  Could just as easily test for its existence instead of trying.